src/components/ActionButton/ActionButton.tsx

```diff
@@ -1,42 +1,51 @@
 import { Button } from '@patternfly/react-core';
-import { TrashIcon, PencilAltIcon, FileIcon } from '@patternfly/react-icons';
+import {
+  TrashIcon,
+  PencilAltIcon,
+  FileIcon,
+  CopyIcon,
+  TagIcon,
+  ExternalLinkAltIcon,
+  LockIcon,
+  FlagIcon,
+} from '@patternfly/react-icons';
 import { useNavigate } from 'react-router-dom';
 
-import styles from './ActionButton.module.css';
-
-const actionIcons = {
+const iconDictionary = {
   create: <FileIcon />,
   edit: <PencilAltIcon />,
   delete: <TrashIcon />,
-};
+  clone: <CopyIcon />,
+  quality: <TagIcon />,
+  external: <ExternalLinkAltIcon />,
+  close: <LockIcon />,
+  mark: <FlagIcon />,
+} as const;
 
 export interface IActionButtonProps {
-  actionType: 'create' | 'edit' | 'delete';
+  iconType: keyof typeof iconDictionary;
+  children?: React.ReactNode;
```

Comment:
use `React.PropsWithChildren` instead, see for example https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/components/EmptyStates/StateCard.tsx#L18